src/utils/password.ts

```diff
@@ -0,0 +1,59 @@
+import {
+  createCipheriv,
+  createDecipheriv,
+  randomBytes,
+  createHash,
+} from "crypto"
+
+import generator from "generate-password"
+
+const ALGORITHM = "aes-256-cbc"
+const { REACT_APP_PASSWORD_SECRET_KEY } = process.env
```

Comment:
As discussed offline, we're going to send passwords in plaintext and rely on transport layer security for now (since this is sent over https) - we'll eventually add extra redundancies and checks, but for the first cut we'll go with this

Modified in https://github.com/isomerpages/isomercms-frontend/pull/1316/commits/5487b17f699e291220087f30b3485344d71087fc!